sale_stock_picking_note/models/stock_picking.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    comments = fields.Text(
+        string="Picking Comments", related="sale_id.picking_comments"
```

Comment:
Maybe that the warehouse user needed to add some public comments as well?

Reply:
Yes, this shouldn't be a related field, but something that is propagated from sales order to picking.